api/sspanel/sspanel.go

```diff
@@ -38,6 +39,9 @@ type APIClient struct {
 	LocalRuleList       []api.DetectRule
 	LastReportOnline    map[int]int
 	access              sync.Mutex
+	eTagForNodeInfo     string
```

Comment:
You can use eTag map[string]string for store eTags. And the controller should determine the error for eTag.